website/docs/r/federated_settings_org_role_mapping.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: mongodbatlas_cloud_federated_settings_org_role_mapping"
+sidebar_current: "docs-mongodbatlas-resource-federated-settings-org-role-mapping"
+description: |-
+    Provides an Federated settings Role Mapping Resource.
+---
+
+# mongodbatlas_cloud_federated_settings_org_role_mapping
+
+`mongodbatlas_cloud_federated_settings_org_role_mapping` provides an Role Mapping resource. This allows organization role mapping to be created.
+
+## Example Usage
+
+```terraform
+resource "mongodbatlas_cloud_federated_settings_org_role_mapping" "org_group_role_mapping_import" {
+  federation_settings_id = "627a9687f7f7f7f774de306f14"
+  org_id                 = "627a9683e7f7f7ff7fe306f14"
+  external_group_name    = "myGrouptest"
+
+  role_assignments {
+    org_id = "627a9683e7f7f7ff7fe306f14"
+    roles     = ["ORG_MEMBER","ORG_GROUP_CREATOR","ORG_BILLING_ADMIN"]
+  }
+
+  role_assignments {
+    group_id = "628aa20d7f7f7f7f7098b81b8"
+    roles     = ["GROUP_OWNER","GROUP_DATA_ACCESS_ADMIN","GROUP_SEARCH_INDEX_EDITOR","GROUP_DATA_ACCESS_READ_ONLY"]
+  }
+
+  role_assignments {
+    group_id = "628aa20d7f7f7f7f7078b81b8"
+    roles     = ["GROUP_OWNER","GROUP_DATA_ACCESS_ADMIN","GROUP_SEARCH_INDEX_EDITOR","GROUP_DATA_ACCESS_READ_ONLY","GROUP_DATA_ACCESS_READ_WRITE"]
+  }
+}
+```
+
+## Argument Reference
+
+* `federation_settings_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
+* `org_id` - Unique 24-hexadecimal digit string that identifies the connected organization.
+* `external_group_name` - Unique human-readable label that identifies the identity provider group to which this role mapping applies.
+* `role_assignments` - Atlas roles and the unique identifiers of the groups and organizations associated with each role.
+* `group_id` - Unique identifier of the project that owns this Role Mapping Configuration.
+* `roles` - Specifies the Roles that are attached to the Role Mapping.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+* `id` - Unique 24-hexadecimal digit string that identifies this role mapping.
+
+## Import
+
+FederatedSettingsOrgRoleMapping must be imported using federation_settings_id-org_id-role_mapping_id, e.g.
```

Comment:
Does this have to be imported if there are no role mappings yet?  I think you can setup an IdP without a role mapping (hence this also being a POST).  Also if it does require require it again, should be called out at the top.

Reply:
Still open - I don't think this one must be.